canonical/canonical.com
# Map the career locations filter to the supported regions

## Done
Map the career locations filter to the supported regions: [APAC, EMEA, AMER]

## QA
- Go to /careers/all?filter=Finance
- Check the location filters work as expected
- Check other departments also work

## Issue / Card
Fixes https://github.com/canonical/canonical.com/issues/665

## Screenshots
![image](https://user-images.githubusercontent.com/1413534/196747152-326fb119-b89a-4257-958d-21b31ec42f45.png)
